spyder_vim/vim_widget.py

```diff
@@ -79,14 +89,19 @@ def DOLLAR(self, repeat=1):
     def ZERO(self, repeat=1):
         self._move_cursor(QTextCursor.StartOfLine)
 
-    def G(self, repeat):
-        editor = self._widget.editor()
-        editor.go_to_line(repeat)
-        self._widget.update_vim_cursor()
-
+    def G(self, repeat=1):
+        self._move_cursor(QTextCursor.Start)
+    
+    def gg(self, repeat=1):
+        self._move_cursor(QTextCursor.End)
```

Comment:
Is is the same as `G`, where entering a number before `gg` does something different ?